games-action/another_world/additional-files/another_world.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+DATA_PATH=`/bin/finddir B_USER_NONPACKAGED_DATA_DIRECTORY`/another_world
```

Comment:
I wonder, if using `findpaths B_FIND_PATH_DATA_DIRECTORY` instead of `finddir`, and later iterate over all returned locations when checking for the existence of the game files, would be better.
That way, if someone created a HPKG with those files, it'd be found as well, because then they would not be in B_USER_NONPACKAGED_DATA_DIRECTORY, but in a regular /system/data or ~/config/data.

When prompting the user with an alert, we should still suggest B_USER_NONPACKAGED_DATA_DIRECTORY, I think. Though B_SYSTEM_NONPACKAGED_DATA_DIRECTORY would work as well (and if Haiku ever goes multiuser, the game data would only needed to be kept once in "system", and not possibly multiple times for each user).

Reply:
ETA:  `findpath -r another_world B_FIND_PATH_DATA_DIRECTORY` will return the game data path if it exists, or an error message.

Reply:
Uhm, in my case `findpaths -r another_world B_FIND_PATH_DATA_DIRECTORY` returns "/boot/system/data" and I don't have the game data directory there.

Anyway I think that the idea to support a multiuser env is good, but:
1) we need a reliable way to dynamically retrieve both $DATA_PATH (and $SAVE_PATH) while checking for files and when launching the game
2) I think that $SAVE_PATH should still point to ~/config/non-packaged/data/another_world in order to avoid save-games corruption in multiuser systems (when Haiku will goes multiuser).

Please let me know what should I do :)

Reply:
> Uhm, in my case `findpaths -r another_world B_FIND_PATH_DATA_DIRECTORY` returns "/boot/system/data" and I don't have the game data directory there.

I think I have misunderstood the -r parameter. Try with `findpaths -e B_FIND_PATH_DATA_DIRECTORY another_world`. Best pipe through `head -n 1` so you only get the highest prio location if there are more than one (e.g. installed in ~/config/non-packaged/data and the system hierarchy).
The above should reliably return the location of the "another_world" data folder, if it exists.

I agree, $SAVE_PATH should always point to ~/config/non-packaged/data/another_world, so each future user has their own save point.